packages/news-api/src/server.ts

```diff
@@ -11,23 +11,40 @@ const options: cors.CorsOptions = {
     origin: '*',
 };
 
+const invalidLimitRes = JSON.stringify({
+    status: 'error',
+    errorMsg: 'Invalid limit param',
+});
+
 app.use(cors(options));
 
 app.get('/', (_req, res) => {
     res.send(`trezor news api`);
 });
 
 app.get('/posts', (req, res) => {
+    let limitCount;
```

Comment:
If the `limit` query param is omitted, this will be undefined. Not sure how TS didn't flag this.